components/locale.js

```diff
@@ -124,14 +113,10 @@ function requireContentFile(localePath, options) {
   }
 }
 
-function getLocaleAndHost(req) {
-  const host = req ? req.headers.host : window.location.host
-  const [locale, ...rest] = host.split('.')
-  if (supportedLocales.includes(locale)) {
-    return {locale, host: rest.join('.')}
-  } else {
-    return {locale: 'en', host}
-  }
+function getLocaleAndHost() {
+  const locale = process.env.LOCALE
+  const host = 'rc.glamorous.rocks'
```

Comment:
Thanks, I've restructured things further and this isn't needed anymore. Push coming soon :)